deno/wipe_appwrite_bucket/README.md

```diff
@@ -0,0 +1,41 @@
+# 🗑 Wipe Appwrite Bucket
+
+A Deno cloud function to wipe a complete bucket in Appwrite by passing the bucket id as a payload.
+
+_Example input:_
+
+```json
+{
+  "bucketId":"profilePictures"
+}
+```
+
+_Example output:_
+
+```json
+{
+  "success":true
+}
+```
+
+## 📝 Environment Variables
+
+List of environment variables used by this cloud function:
+
+- **APPWRITE_FUNCTION_ENDPOINT** - Endpoint of Appwrite project
+- **APPWRITE_FUNCTION_API_KEY** - Appwrite API Key
+- **APPWRITE_FUNCTION_PROJECT_ID** - Appwrite project ID. If running on Appwrite, this variable is provided automatically.
```

Comment:
FYI, `APPWRITE_FUNCTION_PROJECT_ID` is automatically passed to the runtime so you don't need to set the variables.

Reply:
Yes, that's why I added the comment. Or should I remove the ENV from the README completely?

Reply:
Oh sorry, it's fine